app/src/main/java/com/hyperion/ths/marvel_03/ui/favorite/FavoriteFragmentAdapter.java

```diff
@@ -11,6 +11,8 @@
 import com.hyperion.ths.marvel_03.databinding.ItemFragmentFavoriteBinding;
 import com.hyperion.ths.marvel_03.ui.BaseRecyclerView;
 import com.hyperion.ths.marvel_03.ui.OnItemClickListener;
+import io.reactivex.annotations.NonNull;
```

Comment:
ERROR: Unused import: io.reactivex.annotations.NonNull. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports